benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaConsumerBenchmarks.scala

```diff
@@ -69,7 +69,7 @@ object ReactiveKafkaConsumerBenchmarks extends LazyLogging {
         batch.updated(elem)
 
       }
-      .mapAsync(1) { m =>
+      .mapAsync(3) { m =>
```

Comment:
What if we have `mapAsync(3)` which will call `commitAsync(1)` which fails but then we call `commitAsync(2)` before we receive a callback on failure?